data/hai missions.txt

```diff
@@ -1061,3 +1061,93 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Enlisted by Hai [1]"
+	name "Defend <planet>"
+	description "Defeat a pirate raid on <destination>."
+	to offer
+		"combat rating" > 400
+		"random" < 35
```

Comment:
This is a pretty high offer rate

Reply:
I'll lower it to 25% rather than 35%, I don't want to lower it so much so that it becomes easy to miss however.